rviz_default_plugins/src/rviz_default_plugins/displays/tf/tf_display.cpp

```diff
@@ -287,9 +298,33 @@ void TFDisplay::updateFrames()
 {
   typedef std::vector<std::string> V_string;
   V_string frames = context_->getFrameManager()->getAllFrameNames();
-  std::sort(frames.begin(), frames.end());
 
-  S_FrameInfo current_frames = createOrUpdateFrames(frames);
```

Comment:
I'll note that this is the only caller of `createOrUpdateFrames`, so if we end up removing this we should remove the implementation as well.